src/monodroid/config.xml

```diff
@@ -1,5 +1,9 @@
 <configuration>
-	<dllmap dll="java-interop" target="__Internal" />
+	<dllmap dll="java-interop">
+		<dllentry os="linux"   target="xa-internal-api" />
```

Comment:
so we know the dllmap file support is going away for .net 5/6. is it worth updating this to fix that issue now or do it in a separate PR?

Reply:
I think you're right, especially that the change appears to break stuff for some reason (the library isn't found). We need to migrate to programmatic dllmap setup anyway, that is probably best left for a separate PR